master/buildbot/test/unit/www/test_avatar.py

```diff
@@ -92,3 +102,286 @@ def getUserAvatar(self, email, username, size, defaultAvatarUrl):
         res = yield self.render_resource(rsrc, b'/?email=foo')
         self.assertEqual(res, dict(redirected=b'//www.gravatar.com/avatar/acbd18db4cc2f85ce'
                          b'def654fccc4a4d8?d=retro&s=32'))
+
+githubUsernameSearchReply = {
```

Comment:
Let's prefer snake_case for any new code that is not a public API.

Reply:
Done